From eb561754b551e7f453d692bd4d38b54d2821a191 Mon Sep 17 00:00:00 2001
From: Hira Demirkol <hirayigitdemirkol@gmail.com>
Date: Sat, 14 Oct 2023 17:48:34 +0200
Subject: [PATCH] Added SPIR-V 1.3 support to glslang

---
 cmake/vulkan_shader_compiler.cmake | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cmake/vulkan_shader_compiler.cmake b/cmake/vulkan_shader_compiler.cmake
index acc27b5..8b20d66 100644
--- a/cmake/vulkan_shader_compiler.cmake
+++ b/cmake/vulkan_shader_compiler.cmake
@@ -19,6 +19,8 @@ function(vulkan_compile_shader)
                         COMMAND "${GLS_LANG_VALIDATOR_PATH}"
                         ARGS "-V"
                              "${SHADER_COMPILE_INFILE_FULL}"
+                             "--target-env"
+                             "spirv1.3"
                              "-o"
                              "${SHADER_COMPILE_SPV_FILE_FULL}"
                         COMMENT "Compile vulkan compute shader from file '${SHADER_COMPILE_INFILE_FULL}' to '${SHADER_COMPILE_SPV_FILE_FULL}'."
-- 
2.42.0.windows.2

